src/bench/bench_bitcoin.cpp

```diff
@@ -30,6 +30,9 @@ static void SetupBenchArgs(ArgsManager& argsman)
     argsman.AddArg("-output-csv=<output.csv>", "Generate CSV file with the most important benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-output-json=<output.json>", "Generate JSON file with all benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-sanity-check", "Run benchmarks for only one iteration", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-priority-level=<l1,l2,l3>", strprintf("Run benchmarks of a certain priority level (%s | %s | %s), default: %s", PriorityToString(benchmark::PriorityLevel::LOW),
```

Comment:
I would try to not pollute the flag name with specific parameters. So that it does not depend on the format function output (same behavior as every other flag here and `init.cpp`).